README.md

```diff
@@ -58,20 +58,67 @@ pip install -U dataprep
 
 ## Examples & Usages
 
-The following examples can give you an impression of what dataprep can do:
+The following examples can give you an impression of what Dataprep can do:
```

Comment:
Should this be DataPrep?

Reply:
You're right. I got confused because in the current Readme file in some sections is mentioned "Dataprep" and in others "DataPrep". I'll update the info.

Reply:
Done.